src/js/components/RadioButton/doc.js

```diff
@@ -65,9 +65,14 @@ export const themeDoc = {
   },
   'radioButton.border.width': {
     description: 'The width size of the border of the RadioButton.',
-    type: 'string',
+    type: 'string | {dark: string, light: string}',
```

Comment:
This looks like the type modification is being applied to the wrong property here. I think we want to leave `radioButton.border.width` as it was and instead apply the ` | {dark: string, light: string}` to `radioButton.check.background.color`

```suggestion
    type: 'string',
```